src/runtime_src/core/common/query_requests.h

```diff
@@ -200,9 +200,15 @@ enum class key_type
   aie_metadata,
   graph_status,
   mailbox_metrics,
+
   clock_timestamp,
-  noop
+  ert_sleep,
+  ert_cq_write,
```

Comment:
What does cq stand for?

Reply:
cq means command queue. 

We measure the clock cycle while read/write command queue and CU

Reply:
Got it